### DISCUSSION TITLE
Have you ever heard of Mitosis?

### DISCUSSION THREAD

**Discussion by louiss0** [AgnosticUI/agnosticui#244] [Ideas]
I have heard of a tool called Mitosis. It's a UI Library tool that can allow you to create an output using multiple UI Libraries. 
If you are skeptical you can https://github.com/builderio/mitosis. You can even talk to @Steve8708  about it. This tool may help you

**Comment by roblevintennis** [comment]
I'm closing out legacy discussions as the v2 rewrite is almost ready and is a complete rewrite overhaul so discussions are no longer relevant at this point.
